### ISSUE TITLE
Update documentation to add explicit instructions for installation

### ISSUE THREAD

**Issue by timmyomahony** [Styria-Digital/django-rest-framework-jwt#60]
Please add explicit instructions to add both `rest_framework_jwt` **and `rest_framework_jwt.blacklist`** to your `INSTALLED_APPS`.

```
INSTALLED_APPS = [
  'rest_framework_jwt',
  'rest_framework_jwt.blacklist'
]
```

I've just spent hours tracking down an issue that was arising when running my tests. I was getting a red-herring error message when setting up my tests:

> psycopg2.errors.UndefinedTable: relation "user_customuser" does not exist

[Looking online](https://stackoverflow.com/questions/56355516/how-to-resolve-the-psycopg2-errors-undefinedtable-relation-auth-user-does-no), most people with this issue related it to having missing migrations. I checked and double-checked all my migrations for my apps and they were all there.

It was only after I ran my tests with `-v 3` did I realise the issue was happening after trying to set up `rest_framework_jwt`: 

```
...
Running pre-migrate handlers for application team
Running pre-migrate handlers for application user
Running pre-migrate handlers for application volume
Synchronizing apps without migrations:
  Creating tables...
    Creating table rest_framework_jwt_blacklistedtoken
    Running deferred SQL...
Traceback (most recent call last):
  File "/Users/timmyomahony/.virtualenvs/ci-api/lib/python3.7/site-packages/django/db/backends/utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.UndefinedTable: relation "user_customuser" does not exist
```

The problem was that I didn't know about the `blacklist` app within the project, therefore it wasn't included in my `INSTALLEDAPPS`. Regardless, Django was looking for the table `rest_framework_jwt_blacklistedtoken` and was crashing everything.
